source/java/quickstart/index.markdown

```diff
@@ -110,18 +112,24 @@ The `Client` object is what allows you to communicate with Stormpath.
 
     import com.stormpath.sdk.client.Clients;
     import com.stormpath.sdk.client.Client;
+    import com.stormpath.sdk.client.ClientBuilder;
     import com.stormpath.sdk.api.ApiKeys;
     import com.stormpath.sdk.api.ApiKey;
 
     String path = System.getProperty("user.home") + "/.stormpath/apiKey.properties";
```

Comment:
This isn't necessary anymore if they downloaded their key to the default (expected) location. 

I wonder if we should leave out the Google App Engine and API Key building code - it makes the code larger and less readable.  Additionally, because they just downloaded the API Key above, we should be able to assume it is still there.

Thoughts @marioan ?  @omgitstom ?